src/ensembl/production/metadata/updater.py

```diff
@@ -501,13 +501,13 @@ def new_assembly_sequence(self):
 
             # If the name does not match normal accession formating, then use that name.
             name = None
-            if re.match('^[a-zA-Z]+\d+\.\d+', data[0]):
+            if re.match('^[a-zA-Z]+\\d+\.\\d+', data[0]):
```

Comment:
Are you sure about this?
Wouldn't this match "blabla\d.\dddddd" ?

Reply:
This update is meant to fix this warnings in tests:
```src/ensembl/production/metadata/updater.py:504
  /builds/ensembl-production/ensembl-metadata-api/src/ensembl/production/metadata/updater.py:504: DeprecationWarning: invalid escape sequence \.
    if re.match('^[a-zA-Z]+\\d+\.\\d+', data[0]):
src/ensembl/production/metadata/updater.py:510
  /builds/ensembl-production/ensembl-metadata-api/src/ensembl/production/metadata/updater.py:510: DeprecationWarning: invalid escape sequence \.
    if data[1] is not None and re.match('^[a-zA-Z]+\\d+\.\\d+', data[1]):
-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
```